src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/RequestBox/partials/RequestBoxItem.tsx

```diff
@@ -49,7 +49,11 @@ const RequestBoxItem: FC<RequestBoxItemProps> = ({
         }
       }}
     >
-      <span className="text-sm">
+      <span
+        className={clsx('text-sm', {
+          'hover-group:underline': isMotionFailed || !isSingleItem,
```

Comment:
Hey @CzarekDryl shouldn't this be `group-hover` ?

Reply:
@rumzledz My mistake 😢 thanks for catching this